server/models/DB.sql

```diff
@@ -14,15 +14,15 @@ CREATE TABLE users (
   password VARCHAR (100) NOT NULL,
   profile_pic VARCHAR DEFAULT 'userProfile.png',
   company VARCHAR (50) DEFAULT '',	
```

Comment:
`company` and `is_company` should be together in the same table. Consider which table would best fit the information

Reply:
now both are in user table